string_scrap.js

```diff
@@ -1,31 +1,43 @@
+//1
 let word = 'PURSUIT';
 console.log(word[0]);
 
+//2
 let str = 'PURSUIT';
 console.log(str.length);
 
+//3
 let name = 'PURSUIT';
 let lastCHar = name.charAt(name.length - 1);
 console.log(lastCHar);
 
+//4
 let name1 = 'jimmy';
 let lastCHar2 = name1.charAt(name1.length - 1);
 console.log(lastCHar2);
 
+//5
 let drEvil = '10';
 if (drEvil[5] === '0' || drEvil [2] === '0') {
   console.log('pinky');
 } else {
   console.log('10 dollars');
 }
 
+//6
 let strc = 'train';
 let stru = 'box';
 if (strc = 'box' || 'train') {
   console.log(stru + 'ing');
 } else
 console.log(strc + 'ing');
 
+//7
```

Comment:
The problem was asking to get rid of last word which in this case is "birthday". Your output should be "go shawty its ya". Please review strings lesson and look into slice method. Working answer looks like this: 

let h = "I'm happy to be learning JavaScript";
let words = h.split(" ");
let withOutLast = words.slice(0, words.length - 1);
let combined = withOutLast.join(" ");

Run, study, analyze above code and try to learn what slice method does. It is an important method to learn.